### DISCUSSION TITLE
Docs in PDF for offline reading

### DISCUSSION THREAD

**Discussion by shinokada** [colinhacks/zod#1758] [Show and tell]
I just created the Zod docs in pdf for offline reading. 
You can find it out from https://read-doc-download-pdf.vercel.app/#src=/docs/frameworks/zod/en/index.html.
You need to print it out as PDF.
I just needed it for myself but I thought someone may want to read it offline as well, so here you go.

**Comment by JacobWeisenburger** [comment]
This is really cool 😎

Does this get auto updated when the readme is updated?

Also, this doesn't really seem like an issue. Can I move it to the show and tell discussions?

**Comment by shinokada** [comment]
Yes, please move to discussion.
Since I don't have an access to repo, I have to update it manually once a week.
